kore/src/Kore/Internal/MultiOr.hs

```diff
@@ -66,15 +66,12 @@ import Prelude.Kore hiding (
 
 -- | 'MultiOr' is a Matching logic or of its children
 
-{- TODO (virgil): Make 'getMultiOr' a non-empty list ("Data.NonEmpty").
+{-
 
 An empty 'MultiOr' corresponding to 'Bottom' actually discards information about
```

Comment:
I'm not 100% sure, but I think we discussed that this paragraph isn't really correct either, right?

Reply:
Yes, this entire comment can be removed.